__tests__/Unit/Components/Tasks/Card.test.tsx

```diff
@@ -84,4 +84,111 @@ describe("Task card", () => {
 
     expect(getByText(DEFAULT_PROPS.content.status)).toBeInTheDocument();
   });
+
+  it("Should render 'Close task' button when parent issue is closed", function () {
+		const PROPS = {
+			...DEFAULT_PROPS,
+			content: {
+				...DEFAULT_PROPS.content,
+				github: {
+					issue: {
+						closedAt: "2023-04-02T17:31:50",
+						status: "closed",
+						id: 12278,
+					},
+				},
+			},
+		};
+
+		const screen = render(<Card {...PROPS} />);
+
+		const closeTaskBtn = screen.getByRole("button", {
+			name: /Close the task/i,
+		});
+		expect(closeTaskBtn).toBeInTheDocument();
+	});
+
+	it("Should not render 'Close task' button when parent issue is closed but the task was already completed", function () {
+		const PROPS = {
+			...DEFAULT_PROPS,
+			content: {
+				...DEFAULT_PROPS.content,
+        status: "Completed",
+				github: {
```

Comment:
nitpick : formatting

Reply:
Couldn't format this file since it would show formatting changes. 
I think an issue and PR has been raised for formatting and ESlint.

Reply:
okay